mmedit/models/editors/disco_diffusion/clip_ext.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import clip
+import open_clip
```

Comment:
I think `try_import` is a better way to import these modules.

Reply:
True.